package/Runtime/Scripts/Internal/NimbusAdUnit.cs

```diff
@@ -2,6 +2,9 @@
 using UnityEngine;
 
 namespace Nimbus.Runtime.Scripts.Internal {
+
+	public delegate void DestroyAdDelegate();
```

Comment:
I'm likely to change how this is done to be more C# uniform. Just FYI

Reply:
All good.